### PR TITLE
Dry Run Support in Designs And Applications

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

Add Dry Run Support in :
- Applications 
- Patterns

This PR fixes 

https://github.com/meshery/meshery/assets/65964225/ab8530c0-c7b8-440b-a8a0-08d50c75facd

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/api/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
```
Comment (leecalcote) [inline ui/api/index.js:1]: @aabidsofi19 make sure that you highlight this sort of pervasive change prominently in the PR description.

What is this for and what does it do?

File: ui/components/DryRunComponent.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
+import React from "react";
```
Comment (leecalcote) [inline ui/components/DryRunComponent.js:1]: Please organize the components into respective folders. Create new folders as needed and move components around as you see fit.

Comment (aabidsofi19) [reply ui/components/DryRunComponent.js:1]: The Patterns/Applications components were in root , so put it there also . i will create a new folder for this .

File: ui/api/index.js (hunk 1 above)
Comment (aabidsofi19) [reply ui/api/index.js:1]: sorry , forgot we were not using axios in meshery ui  , although it was in dependencies  . Btw this is the axios instance we use for data fetching (in  meshmap also in meshery-cloud ) but seems not in meshery ui . should i remove it .  @leecalcote

File: ui/components/DryRunComponent.js (hunk 2 above)
Comment (leecalcote) [reply ui/components/DryRunComponent.js]: Thank you much. Let's break those old bad habits.

**Pull request merged**